### PR TITLE
When we trigger /v1/regard-application service , callback /v1/redirect-service-request-information to regarded application is returning 400 Bad request error

### LINKED ISSUE #236: When we trigger /v1/regard-application service , callback /v1/redirect-service-request-information to regarded application is returning 400 Bad request error
**Problem Description :**

When we trigger /v1/regard-application , it will send a callback /v1/redirect-service-request-information to regarded application with it's own detalis.

**Root Cause :**
In implementation , the correct formulation of address attribute in request body is not present.

### PR DESCRIPTION
**Pull request by vanithavalluripalli9**
Fixes #236

### CODE REVIEW

File: server/service/individualServices/PrepareForwardingAutomation.js [hunk 1]
```diff
@@ -48,3 +49,11 @@ exports.OAMLayerRequest = function (uuid) {
         }
     });
 }
+
+async function getLocalAddress() {
+            let localAddress = await TcpServerInterface.getLocalAddress();
```
Comment (vanithavalluripalli9) [inline server/service/individualServices/PrepareForwardingAutomation.js:54]: we can add this method in AP as this method might be used by many applications.

**Comment by PrathibaJee** [comment]
reviewer is @DanaSunal

File: server/service/individualServices/PrepareForwardingAutomation.js (hunk 1 above)
Comment (DanaSunal) [reply server/service/individualServices/PrepareForwardingAutomation.js:54]: Agreed, just add another condition for checking domain name.

Comment (vanithavalluripalli9) [reply server/service/individualServices/PrepareForwardingAutomation.js]: Moved the method AP . For domain name the format would be the same as fetched local address so didn't add the condition. Thank you.

**Review by DanaSunal — APPROVED** [review]

**Pull request merged**